shared_memory_client.py

```diff
@@ -0,0 +1,170 @@
+from pickle import loads, dumps
+from tcp_communication import TCP_Connection
+from messages import Request, Response
+import SharedMemoryWrapper
+
+
+class SharedMemoryClient:
+    BUFFER_SIZE = 1024
```

Comment:
We have messages passed via shared memory that are way larger than 1024 bytes. Maybe make this size configurable?

Reply:
good idea
**done!**